script.js

```diff
@@ -1,4 +1,228 @@
+  var outputMessage= `The computer chose ${compChoiceFormatted}<br> You chose ${playerChoiceFormatted}<br><br>${loseMessage}<br><br> ${gameStatus}`
+  return outputMessage
+  }
+};
+//Function for Reverse Game mode
+var reverseGameMode = function (playerChoice, compChoice, playerChoiceFormatted, compChoiceFormatted){
+  totalGamesPlayed = totalGamesPlayed+1
+if (playerChoice == compChoice){
+  totalDraw = totalDraw+1
+  var gameStatus = scoreBoard(userName, playerScore, compScore, totalGamesPlayed,totalDraw)
+  var outputMessage = `It's a draw! 😃 <br> You chose ${playerChoiceFormatted} <br> The Computer chose ${compChoiceFormatted}<br><br> ${gameStatus}`
+  return outputMessage
+  }
+if ((playerChoice == SCISSORS && compChoice == PAPER) || (playerChoice == PAPER && compChoice == STONE) || (playerChoice == STONE && compChoice == SCISSORS )){
+  compScore =compScore+1;
+  var gameStatus = scoreBoard(userName, playerScore, compScore, totalGamesPlayed,totalDraw)
+  var outputMessage= `The computer chose ${compChoiceFormatted}<br> You chose ${playerChoiceFormatted}<br><br>${loseMessage}<br><br> ${gameStatus}`
+  return outputMessage
+  }
+if ((playerChoice == SCISSORS && compChoice == STONE)|| (playerChoice == PAPER && compChoice == SCISSORS) || (playerChoice == STONE && compChoice == PAPER) ){
+ playerScore = playerScore+1;
+ var gameStatus = scoreBoard(userName, playerScore, compScore, totalGamesPlayed,totalDraw)
+ var outputMessage = `The computer chose ${compChoiceFormatted}<br> You chose ${playerChoiceFormatted}<br><br>${winMessage}<br><br> ${gameStatus}`
+  }
+};
+//Function for Korean Game mode
+var koreanGameMode = function (playerChoice, compChoice, playerChoiceFormatted, compChoiceFormatted){
+if (playerChoice == compChoice){
+  winningPlayer = currentWinner
+  var outputMessage = `The computer chose ${compChoiceFormatted}<br> You chose ${playerChoiceFormatted}<br> It's a Draw<br> Current Winner is: ${winningPlayer}<br>${winningPlayer} wins!`
+  return outputMessage
+  }
+if ((playerChoice == SCISSORS && compChoice == PAPER) || (playerChoice == PAPER && compChoice == STONE) || (playerChoice == STONE && compChoice == SCISSORS )){
+  currentWinner = userName
+  var outputMessage = `The computer chose ${compChoiceFormatted}<br> You chose ${playerChoiceFormatted}<br><br>${winMessage}<br>${currentWinner}: "muk-jji-ppa!`
+  return outputMessage
+  }
+if ((playerChoice == SCISSORS && compChoice == STONE)|| (playerChoice == PAPER && compChoice == SCISSORS) || (playerChoice == STONE && compChoice == PAPER) ){
+  currentWinner = 'Computer';
+  var outputMessage= `The computer chose ${compChoiceFormatted}<br> You chose ${playerChoiceFormatted}<br><br>${loseMessage}<br>${currentWinner}: "muk-jji-ppa!"`
+  return outputMessage
+  }
+};
+//Function for Computer Game Mode
+var computerGameMode = function (playerChoice, compChoice, playerChoiceFormatted, compChoiceFormatted){
```

Comment:
Not super necessary, but i would recommend you start making a habit of using console.log()s and comments in your code so that when we come back to out project sometime in the future, it'd be easier for us to pick up from where we left